src/webgl/p5.RendererGL.js

```diff
@@ -2545,4 +2557,24 @@ p5.prototype._assert3d = function (name) {
 
 p5.RendererGL.prototype.tessyVertexSize = 12;
 
+p5.RendererGL.prototype.background = function (...args) {
+  if (args.length === 1 && args[0] instanceof p5.Image) {
```

Comment:
There are some other image-like things that we probably want to work here, e.g. `p5.Graphics` or `p5.MediaElement` (for a webcam capture) or `p5.Texture` or `p5.Framebuffer`. (Maybe at that point it makes sense to flip the logic to be, if the argument is NOT a string or color or number?)